api/src/main/java/org/pmiops/workbench/api/InstitutionController.java

```diff
@@ -66,6 +70,24 @@
     return ResponseEntity.ok(response);
   }
 
+  /**
+   * This API is publicly-accessible since it is called during account creation.
+   *
+   * @return
+   */
+  public ResponseEntity<CheckEmailResponse> checkEmail(final String shortName, final String email) {
```

Comment:
I generally tend to avoid primitive return types for API responses where possible. It dramatically reduces the additional effort for someone to add another response field in the future, which I've found to be a maintenance win.